tests/std/tests/P0674R1_make_shared_for_arrays/test.cpp

```diff
@@ -599,6 +615,12 @@ void test_allocate_shared_array_unknown_bounds() {
     test_allocate_init_destruct_order<ReportAddress[][2][2]>(2u); // success multidimensional
 
     test_allocate_init_destruct_order<ReportAddress[][3][3]>(3u); // failure multidimensional
+
+    allocator<int> a8;
+    shared_ptr<int[7]> p8 = allocate_shared<int[7]>(a8, 0);
+    for (int i = 0; i < 7; ++i) {
+        assert(p8[i] == 0);
+    }

```

Comment:
I believe that this should be testing `allocate_shared<int[]>` (unknown bounds).

Reply:
Fixed

Reply:
I didn't see this in your commits (unsaved editor changes?) so I went ahead and fixed it.